src/detectors/ZDC/ZDC.cc

```diff
@@ -88,14 +91,154 @@ extern "C" {
+             "EcalFarForwardZDCClusterAssociations"}, // edm4eic::MCRecoClusterParticleAssociation
+            {
+              .energyWeight = "log",
+              .sampFrac = 1.0,
+              .logWeightBase = 6.2,
+              .enableEtaBounds = false,
+            },
+            app   // TODO: Remove me once fixed
+          )
+        );
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterHitDigi_factory>(
+          "HcalFarForwardZDCRawHits", {"HcalFarForwardZDCHits"}, {"HcalFarForwardZDCRawHits"},
+          {
+            .tRes = 0.0 * dd4hep::ns,
+            .capADC = 32768,
+            .dyRangeADC = 800 * dd4hep::MeV,
+            .pedMeanADC = 400,
+            .pedSigmaADC = 10,
+            .resolutionTDC = 10 * dd4hep::picosecond,
+            .corrMeanScale = 1.0,
+          },
+          app   // TODO: Remove me once fixed
+        ));
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterHitReco_factory>(
+          "HcalFarForwardZDCRecHits", {"HcalFarForwardZDCRawHits"}, {"HcalFarForwardZDCRecHits"},
+          {
+            .capADC = 32678,
+            .dyRangeADC = 800. * dd4hep::MeV,
+            .pedMeanADC = 400,
+            .pedSigmaADC = 10,
+            .resolutionTDC = 10 * dd4hep::picosecond,
+            .thresholdFactor = 0.0,
+            .thresholdValue = -100.0,
+            .sampFrac = 1.0,
+            .readout = "HcalFarForwardZDCHits",
+            .layerField = "layer",
+            .sectorField = "system",
+          },
+          app   // TODO: Remove me once fixed
+        ));
+
+        app->Add(new JOmniFactoryGeneratorT<HEXPLIT_factory>(
+          "HcalFarForwardZDCSubcellHits", {"HcalFarForwardZDCRecHits"}, {"HcalFarForwardZDCSubcellHits"},
+          {
+            .MIP = 472. * dd4hep::keV,
+            .Emin_in_MIPs=0.1,
+            .tmax=320 * dd4hep::ns,
+          },
+          app   // TODO: Remove me once fixed
+        ));
+
+        double side_length=31.3 * dd4hep::mm;
+        app->Add(new JOmniFactoryGeneratorT<ImagingTopoCluster_factory>(
+            "HcalFarForwardZDCImagingClusterContributions", {"HcalFarForwardZDCSubcellHits"}, {"HcalFarForwardZDCImagingClusterContributions"},
+            {
+                .neighbourLayersRange = 1,
+                .localDistXY = {0.76*side_length, 0.76*side_length*sin(M_PI/3)},
+                .layerDistEtaPhi = {17e-3, 18.1e-3},
+                .sectorDist = 10.0 * dd4hep::cm,
+                .minClusterHitEdep = 100.0 * dd4hep::keV,
+                .minClusterCenterEdep = 11.0 * dd4hep::MeV,
+                .minClusterEdep = 11.0 * dd4hep::MeV,
+                .minClusterNhits = 10,
+            },
+            app   // TODO: Remove me once fixed
+        ));
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterIslandCluster_factory>(
+          "HcalFarForwardZDCIslandClusterContributions", {"HcalFarForwardZDCSubcellHits"}, {"HcalFarForwardZDCIslandClusterContributions"},
```

Comment:
```suggestion
          "HcalFarForwardZDCIslandProtoClusters",
          {"HcalFarForwardZDCSubcellHits"},
          {"HcalFarForwardZDCIslandProtoClusters"},
```
IIRC that's the naming convention - `IslandProtoClusters`